File: Modul-01/python/PRAK107-2310817210009-MuhammadIbnuSina.py
# Input panjang sisi segitiga
input_sisi_str = input(
    "Diketahui : \nPanjang sisi segitiga berturut-turut adalah ")

sisi_list = input_sisi_str.replace("dan", ",").split(',')
sisi_list = [sisi.strip() for sisi in sisi_list]

# Input keliling tanah
keliling_tanah = float(input("Keliling Tanah Pak Dengklek adalah "))

# Input harga tanah per meter
harga_per_meter = float(input("Harga Tanah Per Meter adalah "))

# Menghitung biaya yang diperlukan
biaya = keliling_tanah * harga_per_meter

# Menampilkan hasil
print(
    f"Jawaban : \nBiaya yang diperlukan Pak Dengklek adalah : Rp {int(biaya)}")

File: Modul-01/c/PRAK107-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    float sisi1, sisi2, sisi3;
    float keliling_tanah, harga_per_meter, biaya;

    // Input panjang sisi segitiga
    printf("Diketahui : \nPanjang sisi segitiga berturut-turut adalah ");
    scanf("%f, %f, dan %f", &sisi1, &sisi2, &sisi3);

    // Input keliling tanah
    printf("Keliling Tanah Pak Dengklek adalah ");
    scanf("%f", &keliling_tanah);

    // Input harga tanah per meter
    printf("Harga Tanah Per Meter adalah ");
    scanf("%f", &harga_per_meter);

    // Menghitung biaya yang diperlukan
    biaya = keliling_tanah * harga_per_meter;

    // Menampilkan hasil
    printf("Jawaban : \nBiaya yang diperlukan Pak Dengklek adalah : Rp %.0f\n", biaya);

    return 0;
}
